scan.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright The CrowdStrike Community
+# See License for more info
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+parse_args() {
+  local opts=""
+  while (( "$#" )); do
+    case "$1" in
+      -u|--clientid)
+        if [[ ${#2} -gt 2 ]] ; then
```

Comment:
@isimluk I feel like there is a better solution here for handling both `-u|--clientid` flags and for the case when `--clientid` is used but empty which could happen with actions.yml.